tests/setup.py

```diff
@@ -255,10 +256,7 @@ def insert_characters(self, characters: str) -> int:
 
     def tearDown(self) -> 'Generator':
         yield from self.await_boilerplate_end()
-        super().tearDown()
-
-    def doCleanups(self) -> 'Generator':
-        # restore the user's configs
         close_test_view(self.view)
+        # restore the user's configs
         remove_config(self.config)
-        yield from super().doCleanups()
+        super().tearDown()
```

Comment:
`doCleanups()` no longer used - can I know the specific reasons for this change?

I don't exactly know the differences between those two but I think `doCleanups` is called when something throws while the other doesn't so that could make a difference with `failfast` disabled.

Reply:
Reverted.